unittests/files/htmldocument.ts

```diff
@@ -0,0 +1,2 @@
+const d = new HTMLDocument()
+const result = d instanceof HTMLDocument
```

Comment:
`new HTMLDocument()` should ideally be prevented since it's illegal, I guess this is enough:

```suggestion
document instanceof HTMLDocument
```